app/src/main/java/org/thoughtcrime/securesms/notifications/DefaultMessageNotifier.kt

```diff
@@ -374,8 +394,21 @@ class DefaultMessageNotifier(
     ) {
         Log.i(TAG, "sendMultiThreadNotification()  signal: $signal")
 
-        val builder = MultipleRecipientNotificationBuilder(context, getNotificationPrivacy(context))
         val notifications = notificationState.notifications
+        val existingNotifications = ServiceUtil.getNotificationManager(context).activeNotifications
+        val existingContent = existingNotifications.find { it.id == SUMMARY_NOTIFICATION_ID }?.notification?.extras?.getString("content_hash")
```

Comment:
Should this use the constant instead?
```suggestion
        val existingContent = existingNotifications.find { it.id == SUMMARY_NOTIFICATION_ID }?.notification?.extras?.getString(CONTENT_HASH)
```

Reply:
Fixed